apps/nextjs-website/src/lib/strapi/__tests__/tutorialList.test.ts

```diff
@@ -0,0 +1,105 @@
+import * as E from 'fp-ts/lib/Either';
+import { guideImageJson } from './guides.test';
```

Comment:
We could create a file dedicated to test's features like images, icons, products, meta, etc.